demisto_sdk/__main__.py

```diff
@@ -2082,6 +2079,9 @@ def convert(config, **kwargs):
     "-e", "--examples",
     help="Integrations: path for file containing command examples."
          " Each command should be in a separate line.")
+@click.option(
+    "-a", "--append",
+    help="Append generated test file to the existing.", is_flag=True)
```

Comment:
1. `if exesting` - does this refer to `<integration_name>_test.py`?
2. does it overwrite user-written UT? if so, please mention it (`instead of overwriting`).